webapp/components/admin_console/image_settings.jsx

```diff
@@ -130,7 +130,7 @@ export default class ImageSettings extends AdminSettings {
                     label={
                         <FormattedMessage
                             id='admin.image.previewWidthTitle'
-                            defaultMessage='Preview Width:'
+                            defaultMessage='Image Preview Width:'
```

Comment:
Is this just Image Preview Width or does this also define the dimensions of file previews--for example, code snippet previews? 

If not, where are those defined?